Add HTTP Timeout value to App Service quota

mozlight0302 opened the pull request:
App Service disconnects an HTTP connection with status code 500 if App Service does not respond within 230 seconds, and we cannot configure this value.
See David's answer.
https://social.msdn.microsoft.com/Forums/en-US/17305ddc-07b2-436c-881b-286d1744c98f/503-errors-with-large-pdf-file?forum=windowsazurewebsitespreview

"Why my Web Apps disconnects HTTP request in 230 seconds?", this question is posted frequently on some social sites and CSS often got such a question.
Thus I added HTTP Timeout value as App Service quota clearly.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Add HTTP Timeout value to App Service quota

App Service disconnects an HTTP connection with status code 500.
So I added HTTP Timeout value to App Service quota.
